sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -1565,6 +1567,42 @@ pi_result piDeviceGetInfo(pi_device Device, pi_device_info ParamName,
     }
     return ReturnValue(Supported);
   }
+
+    // intel extensions for GPU information
+  case PI_DEVICE_INFO_PCI_ADDRESS: {
+    if (getenv("ZES_ENABLE_SYSMAN") == nullptr) {
```

Comment:
The env var should be documented in the sycl/doc/EnvironmentVariables.md

Reply:
Done.

Reply:
We are doing similar for enabling of validation layer in Level-Zero loader, search for `setEnvVar`

Reply:
Yes, doing it before zeInit() relieves the user of this burden.

Reply:
> Yes, doing it before zeInit() relieves the user of this burden.

But enabling it always (before zeInit) maybe an extra overhead or otherwise not safe.

Reply:
ZES_ENABLE_SYSMAN needs to be set at the beginning of the application by the user, or before zeInit() in the plugin.  Users will not know to do this, or will forget.  It is better to do it in the plugin.

Reply:
I am afraid that doing it *always* (even in majority of programs where PCI query isn't used) in the plugin is an overkill. Let's stick with what you have: users wanting PCI need to set ZES_ENABLE_SYSMAN (we could actually ask them to use something like SYCL_ENABLE_PCI from which we'd set the ZES_ENABLE_SYSMAN).

Reply:
I see what you are saying.
Setting this environment variable must be done at the beginning of the application.  The environment variables are read as part of zeInit().   I will add SYCL_ENABLE_PCI.

Reply:
Added SYCL_ENABLE_PCI.

Reply:
Thanks!